app/src/main/java/org/p2p/wallet/alarmlogger/AlarmErrorsModule.kt

```diff
@@ -24,10 +29,31 @@ object AlarmErrorsModule : InjectionModule {
             )
             retrofit.create()
         }
-        factoryOf(::AlarmErrorsLogger)
+        factory {
+            // check the children size of  AlarmFeatureConverter to check DI is valid
+            val converters: List<AlarmFeatureConverter> = listOf(
```

Comment:
do not be alarms, this is for scalability 

`AlarmFeatureConverter` - is a base interface for all the alarm converters. We already have 7 of them and more will come!

`List<AlarmFeatureConverter>` comes as a field to root logger and has a method that allows to get typed `T: AlarmFeatureConverter` for the requests